lib/ansible/galaxy/token.py

```diff
@@ -87,9 +199,31 @@ def get(self):
 
         data = json.loads(to_text(resp.read(), errors='surrogate_or_strict'))
 
-        # - extract 'access_token'
+        if self.jwks is None and self._jwks_url is not None:
+            jwks_resp = open_url(
```

Comment:
Can this be a CM? I see `jwks_resp.read()` but not `jwks_resp.close()` meaning that we'll probably have a resource leak here.